x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -249,7 +249,7 @@ export const SearchBar: FC<SearchBarProps> = (opts) => {
         console.log('Error trying to track searchbar metrics', err);
       }
 
-      navigateToUrl(url);
+      navigateToUrl(url, { openInNewTab: true });
```

Comment:
It seems this opens the target in a new tab every time. Can we make it so it only does so if the user holds the CMD key? (Or on Windows, holds the CTRL key)

I think the fix for this doesn't need to involve the `navigateToUrl` method or Application Service. Typically the easiest way to support CMD + Click for opening a link in a new tab is to mark it up as an anchor tag and provide an `href` attribute.
```typescript
function ResultLink() {
  const onClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
    // The "default" behavior is to follow the href, which triggers a
    // full page load. The `event.preventDefault` stops the browser
    // from following the `href`.  The `href` would only be followed
    // when the user holds CMD, and would be handled by the browser
    // would open the link in a new tab.
    event.preventDefault();

    // Navigates to the app in the same tab without a full page load
    navigateToUrl(url);
  };
  return (
    <a href={url} onClick={onClick}>
      {title}
    </a>
  );
}
```

However, I'm not sure how you would achieve this with `EuiSelectableTemplateSitewide`. Maybe update the `renderOption` to render options as anchor tags?

Reply:
++ for href. because it's not only the cmd+click behavior but also other features you get with links like link context menu, mouse wheel click, etc.